test/test_sensors.py

```diff
@@ -319,3 +324,115 @@ def test_get_observations_at():
     assert np.allclose(agent_state.position, start_state.position)
     assert np.allclose(agent_state.rotation, start_state.rotation)
     env.close()
+
+
+def test_noise_models_rgbd():
+    DEMO_MODE = False
+    N_STEPS = 100
+
+    config = get_config()
+    config.defrost()
+    config.SIMULATOR.SCENE = (
+        "data/scene_datasets/habitat-test-scenes/skokloster-castle.glb"
+    )
+    config.SIMULATOR.AGENT_0.SENSORS = ["RGB_SENSOR", "DEPTH_SENSOR"]
+    config.freeze()
+    if not os.path.exists(config.SIMULATOR.SCENE):
+        pytest.skip("Please download Habitat test data to data folder.")
+
+    valid_start_position = [-1.3731, 0.08431, 8.60692]
+
+    expected_pointgoal = [0.1, 0.2, 0.3]
+    goal_position = np.add(valid_start_position, expected_pointgoal)
+
+    # starting quaternion is rotated 180 degree along z-axis, which
+    # corresponds to simulator using z-negative as forward action
+    start_rotation = [0, 0, 0, 1]
+    test_episode = NavigationEpisode(
+        episode_id="0",
+        scene_id=config.SIMULATOR.SCENE,
+        start_position=valid_start_position,
+        start_rotation=start_rotation,
+        goals=[NavigationGoal(position=goal_position)],
+    )
+
+    print(f"{test_episode}")
+    env = habitat.Env(config=config, dataset=None)
+
+    env.episode_iterator = iter([test_episode])
+    no_noise_obs = [env.reset()]
+    no_noise_states = [env.sim.get_agent_state()]
+
+    actions = [
+        sample_non_stop_action(env.action_space) for _ in range(N_STEPS)
+    ]
+    for action in actions:
+        no_noise_obs.append(env.step(action))
+        no_noise_states.append(env.sim.get_agent_state())
+    env.close()
+
+    config.defrost()
+
+    config.SIMULATOR.RGB_SENSOR.NOISE_MODEL = "GaussianNoiseModel"
+    config.SIMULATOR.RGB_SENSOR.NOISE_MODEL_KWARGS = habitat.Config()
+    config.SIMULATOR.RGB_SENSOR.NOISE_MODEL_KWARGS.INTENSITY_CONSTANT = 0.5
+    config.SIMULATOR.DEPTH_SENSOR.NOISE_MODEL = "RedwoodDepthNoiseModel"
+
+    config.SIMULATOR.ACTION_SPACE_CONFIG = "pyrobotnoisy"
+    config.SIMULATOR.NOISE_MODEL = habitat.Config()
+    config.SIMULATOR.NOISE_MODEL.ROBOT = "LoCoBot"
+    config.SIMULATOR.NOISE_MODEL.CONTROLLER = "Proportional"
+    config.SIMULATOR.NOISE_MODEL.NOISE_MULTIPLIER = 0.5
```

Comment:
@mathfac These didn't get added into the default config for some reason?  Can you please add them.

Reply:
@erikwijmans because we don't use `NOISE_MODEL` by default.

Reply:
Hi, I just traced down to the definition of the noise model [here](https://github.com/facebookresearch/habitat-sim/blob/master/habitat_sim/agent/controls/pyrobot_noisy_controls.py#L85) in `habitat-sim`. May I know whether the noise model utilized here is the same as described in Section 9 of [this paper](https://arxiv.org/pdf/1912.06321.pdf), which are fitted on real-world data? Thanks!

Reply:
Yes, this is the same noise model (note that it originates from https://github.com/facebookresearch/pyrobot#citation)

Reply:
Got you. Thanks!